claudette/core.py

```diff
@@ -274,6 +276,32 @@ def structured(self:Client,
     tcs = [call_func(o.name, o.input, ns=ns) for o in cts if isinstance(o,ToolUseBlock)]
     return tcs
 
+# %% ../00_core.ipynb
+def is_builtin(tp: type):
+    "Returns True for built in primitive types or containers"
+    return (tp in (str, int, float, bool, complex) or tp is None
+        or getattr(tp, '__origin__', None) is not None)  # Pass through all container types
+
+
+def convert(val: Dict, # dictionary argument being passed in
```

Comment:
```suggestion
def _convert(val: Dict, # dictionary argument being passed in
```

Prefix `_` since this is private. (Or if its meant to be part of the public API, it needs a better name.) In nbdev private symbols aren't added to `__all__`.